### PR TITLE
SDK-4698: Show skipped modules

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/SDK-4698

### CODE REVIEW

File: src/Upgrade/Application/Dto/ReleaseGroupFilterResponseDto.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    /**
+     * @var \ReleaseApp\Infrastructure\Shared\Dto\Collection\ModuleDtoCollection
+     */
+    protected ModuleDtoCollection $filteredModuleCollection;
+
+    /**
+     * @param \ReleaseApp\Infrastructure\Shared\Dto\ReleaseGroupDto $releaseGroupDto
+     * @param \ReleaseApp\Infrastructure\Shared\Dto\Collection\ModuleDtoCollection|null $filteredModuleCollection
+     */
+    public function __construct(ReleaseGroupDto $releaseGroupDto, ?ModuleDtoCollection $filteredModuleCollection = null)
```
Comment (sergeyspryker) [inline src/Upgrade/Application/Dto/ReleaseGroupFilterResponseDto.php:31]: $proposedModuleCollection

**Review by sergeyspryker — CHANGES_REQUESTED** [review]

File: src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php [hunk 2]
```diff
@@ -260,6 +264,46 @@ protected function responseHasWarnings(StepsResponseDto $stepsResponseDto, Relea
...
+     * @param array<\Upgrade\Application\Dto\ReleaseGroupFilterResponseDto> $filterResponseList
+     *
+     * @return bool
+     */
+    protected function shouldDisplayModuleOfferColumn(array $filterResponseList): bool
+    {
+        foreach ($filterResponseList as $filterResponse) {
+            $isEmpty = $filterResponse->getFilteredModuleCollection()->isEmpty();
+            var_dump('!!!!');
+            var_dump($isEmpty);
```
[the remark this answers is not in the document]
Comment (VladislavStrelchenko) [reply src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php:277]: ++

File: src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php [hunk 3]
```diff
@@ -260,6 +264,46 @@ protected function responseHasWarnings(StepsResponseDto $stepsResponseDto, Relea
...
     }
 
+    /**
+     * @param array<\Upgrade\Application\Dto\ReleaseGroupFilterResponseDto> $filterResponseList
+     *
+     * @return bool
+     */
+    protected function shouldDisplayModuleOfferColumn(array $filterResponseList): bool
+    {
+        foreach ($filterResponseList as $filterResponse) {
```
[the remark this answers is not in the document]
Comment (VladislavStrelchenko) [reply src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php:274]: ++

File: src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php [hunk 4]
```diff
@@ -260,6 +264,46 @@ protected function responseHasWarnings(StepsResponseDto $stepsResponseDto, Relea
...
+        $moduleList = [];
+        foreach ($filterResponseList as $filterResponse) {
+            if ($filterResponse->getReleaseGroupDto()->getId() !== $appliedReleaseGroup->getId()) {
+                continue;
+            }
+            foreach ($filterResponse->getFilteredModuleCollection()->toArray() as $moduleOffer) {
+                $moduleList[] = sprintf('%s:%s', $moduleOffer->getName(), $moduleOffer->getVersion());
+            }
+        }
+
```
[the remark this answers is not in the document]
Comment (VladislavStrelchenko) [reply src/Upgrade/Infrastructure/VersionControlSystem/Generator/PullRequestDataGenerator.php:303]: ++

File: src/Upgrade/Application/Dto/ReleaseGroupFilterResponseDto.php (hunk 1 above)
Comment (VladislavStrelchenko) [reply src/Upgrade/Application/Dto/ReleaseGroupFilterResponseDto.php:31]: ++

**Review by DmytroKlymanSpryker — APPROVED** [review]

**Pull request merged**